indico/modules/events/reminders/forms.py

```diff
@@ -33,14 +37,24 @@ class ReminderForm(IndicoForm):
     # Recipients
     recipients = EmailListField(_('Email addresses'), description=_('One email address per line.'))
     send_to_participants = BooleanField(_('Participants'),
-                                        description=_('Send the reminder to all participants/registrants '
-                                                      'of the event.'))
+                                        description=_('Send the reminder to participants/registrants of the event.'))
+
+    forms = IndicoSelectMultipleCheckboxField(_('Filter by registration forms'),
+                                              [HiddenUnless('send_to_participants')],
+                                              description=_('Limit reminders to participants of '
+                                                            'these registration forms.'))
+    tags = IndicoMultipleTagSelectField(_('Filter by tags'), [HiddenUnless('send_to_participants')],
+                                        description=_('Limit reminders to participants with these tags.'))
+    all_tags = BooleanField(_('All tags must be present'), [HiddenUnless('send_to_participants')],
+                            description=_('Participants must have all of the selected tags. '
+                                          'Otherwise at least one of them.'))
     send_to_speakers = BooleanField(_('Speakers'),
                                     description=_('Send the reminder to all speakers/chairpersons of the event.'))
     # Misc
     reply_to_address = SelectField(_('Sender'), [DataRequired()],
                                    description=_('The email address that will show up as the sender.'))
-    message = TextAreaField(_('Note'), description=_('A custom message to include in the email.'))
+    message = TextAreaField(_('Note'), widget=TinyMCEWidget(),
```

Comment:
The one thing I dislike here is that it now *always* shows the WYSIWYG editor... it's very bulky and it encourages people to use HTML formatting a lot. I don't have a good solution for it though.

Reply:
ah hm, maybe easier for now to stick w/ the HTML one since we use it in mayn other places as well